samples/net/udp/prj.conf

```diff
@@ -0,0 +1,23 @@
+#
+# Copyright (c) 2023 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+# General configurations
+CONFIG_PICOLIBC=y
+CONFIG_LOG=y
+CONFIG_ASSERT=y
```

Comment:
I think we can keep asserts disabled by default in a sample

Reply:
No objections, removed for now.